### PR TITLE
[Doc] Add nnstreamer tutorials

### PR DESCRIPTION
**Pull request by gichan-jang**
- Add NNStreamer tutorials.
 - Change sitemap of the nnstreamer.github.io

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

**Review by dongju-chae — APPROVED** [review]
Looks great :)

File: Documentation/hotdoc/tutorials.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Welcome to the NNStreamer tutorials.
+### Prerequisites
+Before starting the tutorial, you should install GStreamer and NNStreamer in advance.  
+Refer to [here](installing-nnstreamer.md) for detailed installation instructions.  
+If you're a Ubuntu user, the packages needed in each tutorial will be described in the tutorial.  
+So even if you haven't prepared, let's go to tutorial one.
+
+### Basic Tutorials
+Tutorials introduce how to use NNSstreamer step by step.
```
Comment (jaeyun-jung) [inline Documentation/hotdoc/tutorials.md:15]: typo NNSstreamer

File: Documentation/tutorials/tutorial1_playing_video.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Factory Details:
+  Rank                     none (0)
+  Long-name                Colorspace converter
+  Klass                    Filter/Converter/Video
+  Description              Converts video from one colorspace to another
+...
+```
+ - v4l2src: Reads frames from a linux video device.
+ - videotestsrc: Creates a test video stream.
+ - videoconvert: Convert video format.
```
Comment (jaeyun-jung) [inline Documentation/tutorials/tutorial1_playing_video.md:39]: Converts video format.

File: Documentation/tutorials/tutorial1_playing_video.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ - videoscale: Resizes video.
+ - autovideosink: Automatically detectes video sink.
```
Comment (jaeyun-jung) [inline Documentation/tutorials/tutorial1_playing_video.md:41]: detects

File: Documentation/tutorials/tutorial4_query.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: T4. Edge pipeline - Query
+...
+
+# Tutorial 4. Edge pipeline - Query
+Tensor query allows devices which have weak AI computational power to to use resources from higher-performance devices.  
+Suppose you have a device at home with sufficient computing power (server) and a network of lightweight devices connected to it (clients).  
+The client asks the server to handle heavy tasks and receive results from the server. Therefore, there is no need for cloud server by running AI on a local network.
```
Comment (jaeyun-jung) [inline Documentation/tutorials/tutorial4_query.md:8]: receives results

File: Documentation/tutorials/tutorial5_gstreamer_api.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  gst_element_set_state (g_app.pipeline, GST_STATE_NULL);
+  gst_object_unref (element);
+
+  ...
+}
+```
+
+Then let's run the app.
+```
+# If you built tourial 5 yourself, go to the path you specified, and if you installed it using apt, go to /usr/lib/nnstreamer/bin.
```
Comment (jaeyun-jung) [inline Documentation/tutorials/tutorial5_gstreamer_api.md:60]: tourial > tutorial

File: Documentation/tutorials/tutorial5_gstreamer_api.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  ...
+
+  /* start pipeline. Setting pipeline to the PLAYING state. */
+  gst_element_set_state (g_app.pipeline, GST_STATE_PLAYING);
+
+  ...
+  /** pipeline is running */
+  ...
+
+  /** Stop and release the resoureces */
```
Comment (jaeyun-jung) [inline Documentation/tutorials/tutorial5_gstreamer_api.md:50]: resoureces > resources

File: Documentation/tutorials/tutorial5_gstreamer_api.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  gst_element_set_state (g_app.pipeline, GST_STATE_NULL);
+  gst_object_unref (element);
```
Comment (jaeyun-jung) [inline Documentation/tutorials/tutorial5_gstreamer_api.md:52]: gst_object_unref (element); > gst_object_unref (g_app.pipeline);

**Comment by gichan-jang** [comment]
@jaeyun-jung Thanks! Typo and unclear sentences are updated.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**